motionplan/solverFrame.go

```diff
@@ -262,6 +271,9 @@ func (sf *solverFrame) Geometries(inputs []frame.Input) (*frame.GeometriesInFram
 	inputMap := sf.sliceToMap(inputs)
 	sfGeometries := []spatial.Geometry{}
 	for _, fName := range sf.movingFS.FrameNames() {
+		if strings.Contains(fName, "ExecutionFrame") {
```

Comment:
Per @biotinker's comment right above, this will also need to be removed.. I will need to think about how to best accomplish this.